### PR TITLE
Upgrade MRTCore sample apps to use the Project Reunion NuGet.

### PR DESCRIPTION
**Pull request by rohanp-msft**
**How verified:**
- Built the class library.
- Built and ran the C# app.
- Built and ran the C++ app.

### CODE REVIEW

**Review by axelandrejs — APPROVED** [review]

File: MrtCore/winui_desktop_packaged_app/winui_desktop_packaged_app (Package)/winui_desktop_packaged_app (Package).wapproj [hunk 1]
```diff
@@ -61,13 +60,30 @@
...
+    <PkgMicrosoft_ProjectReunion Condition="!Exists($(PkgMicrosoft_ProjectReunion))">$(SolutionDir)packages\Microsoft.ProjectReunion.0.5.0-prerelease\</PkgMicrosoft_ProjectReunion>
+    <PkgMicrosoft_ProjectReunion_WinUI Condition="'$(PkgMicrosoft_ProjectReunion_WinUI)'==''">$([MSBuild]::NormalizeDirectory('$(NuGetPackageRoot)', 'Microsoft.ProjectReunion.WinUI', '0.5.0-prerelease'))</PkgMicrosoft_ProjectReunion_WinUI>
+    <PkgMicrosoft_ProjectReunion_WinUI Condition="!Exists($(PkgMicrosoft_ProjectReunion_WinUI))">$(SolutionDir)packages\Microsoft.ProjectReunion.WinUI.0.5.0-prerelease\</PkgMicrosoft_ProjectReunion_WinUI>
+    <Microsoft_ProjectReunion_AppXReference_props>$([MSBuild]::NormalizeDirectory('$(PkgMicrosoft_ProjectReunion)', 'build'))Microsoft.ProjectReunion.AppXReference.props</Microsoft_ProjectReunion_AppXReference_props>
+    <Microsoft_WinUI_AppX_targets>$([MSBuild]::NormalizeDirectory('$(PkgMicrosoft_ProjectReunion_WinUI)', 'build'))Microsoft.WinUI.AppX.targets</Microsoft_WinUI_AppX_targets>
```
Comment (huichen123) [inline MrtCore/winui_desktop_packaged_app/winui_desktop_packaged_app (Package)/winui_desktop_packaged_app (Package).wapproj:75]: if you look at the latest template, they are no longer needed.

Comment (Scottj1s) [reply MrtCore/winui_desktop_packaged_app/winui_desktop_packaged_app (Package)/winui_desktop_packaged_app (Package).wapproj:75]: Agreed - I know this PR has been completed, but updating all samples wapproj files to pattern after the VSIX templates will enable upgrades, etc

**Comment by zaryaf** [comment]
Do the samples links in our docs need to be updated or is the same URL?

**Comment by rohanp-msft** [comment]
> Do the samples links in our docs need to be updated or is the same URL?
The link under Samples here
https://docs.microsoft.com/en-us/windows/apps/project-reunion/mrtcore/mrtcore-overview
took me here
https://github.com/microsoft/Project-Reunion-Samples/tree/main/MrtCore

So I think we're OK for the URL.

**Comment by zaryaf** [comment]
@mcleanbyron Just noting that we do not need to change any of the samples URL's based on Rohan. Thanks.

**Pull request merged**